src/getItems.js

```diff
@@ -25,7 +25,9 @@ async function getItems(pageObj, pageData, resultsArr, label) {
             const priceExists = (await item.$('.a-color-price')) || null;
             obj.price = priceExists ? await item.$eval('.a-color-price', el => el.innerText) : null;
             obj.url = await item.$eval('div > a.a-link-normal:nth-of-type(1)', url => url.href);
-            obj.thumbnail = await item.$eval('div[class*="_p13n-zg-list-grid-desktop_maskStyle"] > img', url => url.src);
+            // p13n-sc-dynamic-image
+            // const thumbSelector = await item.$('div[class*="_p13n-sc-dynamic-image"]') || await item.$('div[class*="_p13n-zg-list-grid-desktop_maskStyle"] > img') || '';
+            obj.thumbnail = await item.$eval('img', url => url.src);
```

Comment:
Can you just make this infallible? So in case the selector is not there, it doesn't crash? That's how the parsing should mostly be done.

We wrap each into .catch or just rewrite to JQuery